frame/nomination-pools/src/tests.rs

```diff
@@ -2159,6 +2285,184 @@ mod withdraw_unbonded_other {
 			assert!(!BondedPools::<Runtime>::contains_key(1));
 		});
 	}
+
+	#[test]
+	fn partial_withdraw_unbonded_depositor() {
+		ExtBuilder::default().ed(1).build_and_execute(|| {
+			// so event the depositor can leave, just keeps the test simpler.
+			unsafe_set_state(1, PoolState::Destroying).unwrap();
+
+			// given
+			assert_ok!(Pools::unbond_other(Origin::signed(10), 10, 6));
```

Comment:
Depositor should be able to partial unbond

Reply:
should _not_ you mean?

Reply:
Yes, my bad :D

Reply:
Now this test is correct because the depositor is only partially unbondint to a point above `MinCreateBond`. Added a new test to demonstrate the other case as well.